README.md

```diff
@@ -61,7 +61,7 @@ This package adds a `json5_decode()` function which is a drop-in replacement for
 
 ``` php
 $json = file_get_contents('foo.json5');
-$arr = json5_decode($json);
+$arr = json5_decode($json, true);
```

Comment:
I would rather rename the variable to `$data` since the returned value could be anything (including an array, a string, an integer, etc)

Reply:
Updated to use `$data`.